src/app/security/maturity-model/page.tsx

```diff
@@ -0,0 +1,83 @@
+import { PageHeader } from '@/components/PageHeader'
+import { PageLayout } from '@/components/PageLayout'
+import { PageSection } from '@/components/PageSection'
+import { StructuredDataScript } from '@/components/StructuredDataScript'
+
+import { createMetadata } from '@/utils/createMetadata'
+
+import { graphicsData } from '@/data/graphicsData'
+
+import { attributes } from '@/content/pages/maturity-model.md'
+
+import { PATHS } from '@/constants/paths'
+
+import { CoreFunctions } from './components/CoreFunctions'
+import { generateStructuredData } from './utils/generateStructuredData'
+
+const { header, seo } = attributes
+
+export const metadata = createMetadata({
+  seo,
+  path: PATHS.MATURITY_MODEL.path,
+})
+
+export default function MaturityModel() {
```

Comment:
Wondering if we should consider [dynamic import ](https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading) for this page since we're loading a lot of content?

Reply:
<img width="480" alt="Screenshot 2024-08-29 at 12 05 32" src="https://github.com/user-attachments/assets/4f13b9a8-c2be-4d33-8089-c1851e9ee442">

Reply:
Let's discuss this tomorrow with @CharlyMartin